registry/storage/driver/gdrive/gdrive_test.go

```diff
@@ -0,0 +1,135 @@
+package gdrive
+
+import (
+	"os"
+	"testing"
+
+	ctx "github.com/docker/distribution/context"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"github.com/docker/distribution/registry/storage/driver/testsuites"
+	check "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { check.TestingT(t) }
+
+var gdriveDriverConstructor func(rootDirectory string) (storagedriver.StorageDriver, error)
+var skipGdrive func() string
+
+func init() {
+
+	keyFile := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS_FILE")
```

Comment:
Do we need to add a script file in contrib folder that exports environment variable ?